.github/workflows/validate-spec.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# It will check, for each spec file passed, that either the version or the release or both have been updated
+
+# $1 - Branch to diff against
+# $@:2 - Paths to spec files to check
+
+[[ $# -eq 1 ]] && echo "No specs passed to validate"
+git fetch origin $1
+
+for spec in "${@:2}"
+do
+  echo Checking "$spec"
+
+  # Ensure spec can be parsed
+  name=$(rpmspec --srpm  --define "with_check 0" --qf "%{NAME}" -q $spec 2>/dev/null )
```

Comment:
Suggestion: let's have a separate check for `with_check 1`, since we run it both ways.

Reply:
Added a check for with_check 1 condition